src/main/java/it/gov/pagopa/rtd/ms/rtdmsfilereporter/domain/repository/FileReportRepository.java

```diff
@@ -3,8 +3,12 @@
 
 import it.gov.pagopa.rtd.ms.rtdmsfilereporter.domain.model.FileReport;
 import java.util.Collection;
+import org.springframework.transaction.annotation.Transactional;
 
 public interface FileReportRepository {
 
   Collection<FileReport> getReportsBySenderCodes(Collection<String> senderCodes);
+
+  @Transactional
```

Comment:
@and-mora @Transactional doesn't work well if the consistency model of cosmos db is set to "Strong", could you please check?

Reply:
It is "strong" actually, thank you for the tip.